exercises/prime-factors/src/example/kotlin/PrimeFactorCalculator.kt

```diff
@@ -0,0 +1,38 @@
+object PrimeFactorCalculator {
+
+    fun primeFactors(int: Int): List<Int> {
+        val result = mutableListOf<Int>()
+        var remainder = int
+        var divisor: Int = 2
+
+        while (remainder > 1) {
+            while (remainder.rem(divisor) == 0) {
+                result.add(divisor)
+                remainder /= divisor
+            }
+
+            divisor++
+        }
+
+        return result
+    }
+
+
+    fun primeFactors(long: Long): List<Long> {
+        val result = mutableListOf<Long>()
+        var remainder = long
+        var divisor: Long = 2
+
+        while (remainder > 1) {
+            while (remainder.rem(divisor) == 0L) {
+                result.add(divisor)
+                remainder /= divisor
+            }
+
+            divisor++
+        }
+
+        return result
+    }
```

Comment:
You could call the `primeFactors` method with an int cast to a long then map the `list<Long>` to an int via `list.map { it.toInt() }`